src/snippets/index.md

```diff
@@ -0,0 +1,12 @@
+# Snippets
```

Comment:
Wondering if the snippets could be a single page, instead of multiple, given that these are pretty much just an editor? Unless you plan to add more content above or below the editor for each snippet that is. The sidebar would appear as the same if each snippet follows a `##` heading, and the search would pick these up still. The two scenarios on my mind are:

* Each snippet has a bit of prose to it above or below the editor, explaining what it does or giving context, then multiple pages would be fine as pages wouldn't look too empty.
* It's just a headline and the editor, where a single page would feel like "richer" documentation (for the lack of a better word)